examples/Emulator/GaussianProcess/Manifest.toml

```diff
@@ -106,22 +106,22 @@ uuid = "83423d85-b0ee-5818-9007-b63ccbeb887a"
 version = "1.16.1+1"
 
 [[deps.CalibrateEmulateSample]]
-deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
-path = "../../.."
+deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Pkg", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
+git-tree-sha1 = "5a93353fdc751a2116529e907316cc51275b77b8"
```

Comment:
You probably don’t want to make these changes to CES.

Reply:
Thanks for the catch! Reverted in https://github.com/CliMA/CalibrateEmulateSample.jl/pull/177/commits/731abcd7527f843c62a9c74f5a69035e18a2801f.